examples/B-RSS-train-set/README.md

```diff
@@ -4,10 +4,10 @@ Energies in Elemental Boron PRL 2018
 
 #
 * step 0
-frame_select.py -fxyz sps_all.xyz --n 1000 --prefix subset --algo random
+python ../../scripts/frame_select.py -fxyz sps_all.xyz --n 1000 --prefix subset --algo random
```

Comment:
We decided to just use the name of the script, instead of the relative path in the examples,
because we expect people to install it